api/controllers/TOTPController.js

```diff
@@ -42,12 +65,42 @@ module.exports = {
     };
   },
 
-  // Empty endpoint to verify a TOTP token
-  verifyTOTPToken(request, reply) {
-    return reply.response().code(204);
-  },
-
-  // Enables TOTP for current user
+  /*
+   * @api [post] /totp
+   * tags:
+   *   - totp
+   * summary: Enables 2FA for current user.
+   * parameters:
+   *   - name: X-HID-TOTP
+   *     in: header
+   *     description: The TOTP token. Required.
+   *     required: false
+   *     type: string
+   * requestBody:
+   *   description: >-
+   *     Must be an object specifying: `{"method": "app"}`
+   *   required: true
+   *   content:
+   *     application/json:
+   *       schema:
+   *         type: object
+   *         properties:
+   *           method:
+   *             type: string
+   *             required: true
+   *             pattern: '^app$'
```

Comment:
Yes it will be removed, so I'll file a ticket to do the coding work. I didn't want to add non-trivial code changes into these docs branches. HID-2082.

Reply:
#118 contains the fix for this